workers/unity/Packages/com.improbable.gdk.core/Config/ConnectionConfig.cs

```diff
@@ -77,13 +77,26 @@ public ConnectionParameters CreateConnectionParameters()
         /// <exception cref="ConnectionFailedException">
         ///     Thrown if configValue is null or empty.
         /// </exception>
-        protected void ValidateConfig(string configValue, string configName)
+        protected void ValidateString(string configValue, string configName)
         {
-            if (string.IsNullOrEmpty(configValue))
+            ValidateCondition(!string.IsNullOrEmpty(configValue),
```

Comment:
I still have no idea why we have this function.

Reply:
Which one? The old one that checked whether a string was not null/empty or the new one that checks a bool?

Reply:
Not a criticism on this PR.

Reply:
if it's not useful, we should remove it

Reply:
I've removed the two validation methods and refactored the test code.